script.py

```diff
@@ -0,0 +1,41 @@
+import requests
+import pprint
+import config
```

Comment:
Base on pep8 standard and our style guide, the following import groups should be separated by a blank line:

1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.

And within each of these groups, imports should be sorted alphabetically. 

Ref: https://github.com/osu-mist/style-guides/blob/master/Python.md#imports